src/components/Modal.js

```diff
@@ -14,8 +14,10 @@ export default React.createClass({
     onEnter: React.PropTypes.func,
   },
 
-  defaultProps: {
-    isOpen: false,
+  getDefaultProps() {
```

Comment:
eh, it is... I thought it wouldn't appear if I made the second pr in another branch.